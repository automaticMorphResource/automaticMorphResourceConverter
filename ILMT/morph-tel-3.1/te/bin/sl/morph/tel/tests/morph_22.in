1	AstreliyA	unk	
1	wIrAniki	unk	
1	kAswa	unk	
1	xUraMlo	unk	
1	,	unk	
1	samuxraMlo	unk	
1	uMde	unk	
1	I	unk	
1	kaMgArU	unk	
1	xIvi	unk
1	..	unk	
1	AstreliyA	unk	
1	cuttU	unk	
1	uMde	unk	
1	cinna	unk	
1	cinna	unk	
1	xIvulalo	unk	
1	unna	unk
1	"	unk	
1	tAsmAniyA	unk	
1	"	unk	
1	xIvikaMte	unk	
1	kAswa	unk	
1	cinnaxigA	unk	
1	uMtuMxi	unk	
1	.	unk	
