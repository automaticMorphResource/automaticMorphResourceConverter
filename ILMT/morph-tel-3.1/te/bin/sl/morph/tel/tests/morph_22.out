1	AstreliyA	unk	<fs af='AstreliyA,n,,sg,,d,0,0'>|<fs af='AstreliyA,n,,sg,,o,ti,ti'>
2	wIrAniki	unk	<fs af='wIraM,n,,sg,,,ki,ki'>
3	kAswa	unk	<fs af='kAswa,n,,sg,,d,0,0'>|<fs af='kAswa,n,,sg,,o,ti,ti'>
4	xUraMlo	unk	<fs af='xUraM,n,,sg,,,lo,lo'>
5	,	unk	<fs af='&comma;,punc,,,,,,'>
6	samuxraMlo	unk	<fs af='samuxraM,n,,sg,,,lo,lo'>
7	uMde	unk	<fs af='uMdu,v,any,any,any,,e,e'>|<fs af='uMdu,v,any,any,any,,i,i_e'>
8	I	unk	<fs af='I,avy,,,,,,'>|<fs af='I,avy,,,,,,'>
9	kaMgArU	unk	<fs af='kaMgArU,n,,sg,,d,0,0'>|<fs af='kaMgArU,n,,sg,,o,ti,ti'>|<fs af='kaMgAru,n,,sg,,,0,0_V'>
10	xIvi	unk	<fs af='xIvi,n,,sg,,d,0,0'>|<fs af='xIvi,n,,sg,,o,ti,ti'>
11	..	unk	<fs af='&dot;,punc,,,,,,'>
12	AstreliyA	unk	<fs af='AstreliyA,n,,sg,,d,0,0'>|<fs af='AstreliyA,n,,sg,,o,ti,ti'>
13	cuttU	unk	<fs af='cuttU,avy,,,,,0,0_avy'>|<fs af='cuttu,n,,sg,,,0,0_V'>
14	uMde	unk	<fs af='uMdu,v,any,any,any,,e,e'>|<fs af='uMdu,v,any,any,any,,i,i_e'>
15	cinna	unk	<fs af='cinna,n,,sg,,d,0,0'>|<fs af='cinna,n,,sg,,o,ti,ti'>|<fs af='cinna,adj,,,,,0,0_adj'>|<fs af='cinnaM,n,,sg,,o,ti,ti'>
16	cinna	unk	<fs af='cinna,n,,sg,,d,0,0'>|<fs af='cinna,n,,sg,,o,ti,ti'>|<fs af='cinna,adj,,,,,0,0_adj'>|<fs af='cinnaM,n,,sg,,o,ti,ti'>
17	xIvulalo	unk	<fs af='xIvi,n,,pl,,,lo,lo'>
18	unna	unk	<fs af='unna,adj,,,,,0,0_adj'>|<fs af='uMdu,v,any,any,any,,ina,ina'>
19	"	unk	<fs af='&bdquo;,punc,,,,,,'>
20	tAsmAniyA	unk	<fs af='tAsmAniyA,n,,sg,,d,0,0'>|<fs af='tAsmAniyA,n,,sg,,o,ti,ti'>
21	"	unk	<fs af='&bdquo;,punc,,,,,,'>
22	xIvikaMte	unk	<fs af='xIvi,n,,sg,,,kaMteV,kaMteV'>
23	kAswa	unk	<fs af='kAswa,n,,sg,,d,0,0'>|<fs af='kAswa,n,,sg,,o,ti,ti'>
24	cinnaxigA	unk	<fs af='cinna,pn,,sg,,,xi,xi_gA'>
25	uMtuMxi	unk	<fs af='uMdu,v,fn,sg,3,,wA,wA'>
26	.	unk	<fs af='&dot;,punc,,,,,,'>
